### ISSUE TITLE
Simulation doesn't work for imported sub processes in a super BP

### ISSUE THREAD

**Issue by dnsmichi** [Icinga/icingaweb2-module-businessprocess#229]
## Expected Behavior

Simulation with state overrides also works with imported linked BPs.

## Current Behavior

Imported sub BPs don't allow you editing and simulate a state change. You have to navigate back into the main BP which makes the feature kind of hard to use in the "super BP". 

## Steps to Reproduce (for bugs)

1. Create 2 BPs, web and database, root node production with some services in it
2. Create a super BP, with root nodes web and database
3. Import the existing BPs into the new root nodes
4. Unlock editing and try to simulate a state change - this icon doesn't exist in this view

![Screen Shot 2019-04-10 at 16 59 29](https://user-images.githubusercontent.com/382049/55890109-6ba82a00-5bb2-11e9-9625-616c261e0093.png)

5. Navigate back into the database or web BP - you can edit and simulate the BP

![Screen Shot 2019-04-10 at 17 02 18](https://user-images.githubusercontent.com/382049/55890136-76fb5580-5bb2-11e9-888e-66470b90ca0f.png)

## Context

Building multiple layers of BPs.

web.conf
```
### Business Process Config File ###
#
# Owner           : icingaadmin
# AddToMenu       : yes
# Statetype       : soft
#
###################################

production = web-host-1;http & web-host-2;http
display 1;production;production
```

database.conf
```
### Business Process Config File ###
#
# Description     : All database checks for our production cluster
# Owner           : icingaadmin
# AddToMenu       : yes
# Statetype       : soft
#
###################################

production = mysql-host-1;mysql-connection-time & mysql-host-2;mysql-connection-time
display 1;production;production
```

shop.conf
```
### Business Process Config File ###
#
# Owner           : icingaadmin
# AddToMenu       : yes
# Statetype       : soft
#
###################################

database = @database:production
display 1;database;database
web = @web:production
display 1;web;web
```

## Your Environment

* Module version (System - About): 2.2.0

**Comment by nilmerg** [comment]
I've looked into this prior v2.2 but the way it's implemented I was unable to adjust this in time. The simulation feature is tightly connected with how processes are edited. Internally this is separated on a per configuration basis. Not so easy to change.

**Comment by dnsmichi** [comment]
Agreed, it likely requires larger rewrites. In earlier versions of the BP module, is was easier to simulate since the whole 'unlock editing, store config, diff config` mechanism wasn't that complex. That's mostly the reason why I've created this issue, BP simulation is one of the key features imho :)

**Comment by Thomas-Gelf** [comment]
Hint: the toggle used to be labelled "Unlock", this has been changed to "Unlock Editing"

**Comment by nilmerg** [comment]
That doesn't change the fact that it's an additional click (hidden feature), which this issue isn't even about but #226 is. :man_shrugging:

**Comment by Thomas-Gelf** [comment]
@nilmerg: you're right, my comment relates to #226 - sorry for the confusion. Changing that label might help arguing in #226 ;-)

**Comment by MarcusCaepio** [comment]
Hi all,
I am re-using existing processes too and just stepped over this issue. Any plans to work on this in the near future?

**Comment by nilmerg** [comment]
No plans, yet.
